modules/javafx.graphics/src/main/java/com/sun/glass/ui/monocle/EGLPlatform.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+package com.sun.glass.ui.monocle;
+
+public class EGLPlatform extends LinuxPlatform {
+
+    /**
+     * Create an <code>EGLPlatform</code>. If a library with specific native code is needed for this platform,
+     * it will be downloaded now. The system property <code>monocle.egl.lib</code> can be used to define the
+     * name of the library that should be loaded.
+     */
+    public EGLPlatform() {
+        String lib = System.getProperty("monocle.egl.lib");
+        if (lib != null) {
+            LinuxSystem.getLinuxSystem().dlopen(lib, LinuxSystem.RTLD_LAZY | LinuxSystem.RTLD_GLOBAL);
+        }
+    }
+
+    @Override
+    public synchronized AcceleratedScreen getAcceleratedScreen(int[] attributes) throws GLException {
+        return new EGLAcceleratedScreen(attributes);
```

Comment:
Should this method cache the result so it returns a singleton (not sure, but the other `NativePlatform` subclasses, including `Dispman` do)?

Reply:
Good question. In theory (e.g. on the Pi 4), there can be 2 different hardware accelerated screens, but their is no application logic in the higher layers for distributing rendering to 2 different interfaces. 
Hence, it is indeed better to cache the result.